app/models/post.rb

```diff
@@ -1,3 +1,17 @@
 class Post < ApplicationRecord
-  belongs_to :author, class_name: 'User'
+  after_save :post_increment
```

Comment:
✅ Good job using a callback to update the posts counter every time a new post has been saved to database 👍🏻